src/core/Track.cpp

```diff
@@ -1913,7 +1913,7 @@ bool TrackContentWidget::pasteSelection( MidiTime tcoPos, const QMimeData * md,
 
 		TrackContentObject * tco = t->createTCO( pos );
 		tco->restoreState( tcoElement );
-		tco->movePosition( pos );
+		tco->movePosition(pos); // Because we restored the state, we need to move the TCO again.
```

Comment:
Just being a nitpick, but shouldn't it be `( pos )` instead of `(pos)`? (Based on the way it is written two lines above)

Reply:
From what I understood, `(pos)` is actually the way endorsed by our code style guideline (I'm guilt of using spaces inside parenthesis before though, personally I thought they were more readable). On my PRs I've been trying to fix the code style on the lines that are affected by the PR only (that's why the lines above still have `( pos )`), that's so reviewing doesn't get cluttered by all the code style fixes

Reply:
Oh, if that is the case, then no issue. I thought it would be better to keep it consistent, but you've already been doing this, so no objections from me.

Reply:
Soon we should have the clang-format to fix the remaining code style issues, at least those parenthesis spaces should be easily fixable with it :smile: